src/main/java/com/parasoft/demoapp/graphql/RestTemplateUtil.java

```diff
@@ -25,13 +26,14 @@ public static HttpHeaders createHeaders(HttpServletRequest httpRequest) {
     }
 
     public static GraphQLException convertException(Exception e) {
+        log.error(e.getMessage(), e);
```

Comment:
There will be duplicates for printing this here and inside constructor of "GraphQLException".